rootfs/api/tests/test_auth.py

```diff
@@ -248,7 +248,7 @@ def test_passwd(self):
         }
         response = self.client.post(url, submit,
                                     HTTP_AUTHORIZATION='token {}'.format(token))
-        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.status_code, 401)
```

Comment:
If we are going forward with this change can you please make a doc PR to the API reference docs so we can keep it up to date? Thanks!